src/event.js

```diff
@@ -610,8 +610,7 @@ jQuery.event = {
 			event,
 			{
 				type: type,
-				isSimulated: true,
-				originalEvent: {}
```

Comment:
Nuh, that is the thing it wouldn't, see - https://github.com/jquery/jquery/pull/2303/files#r30607068